cohttp-async/src/client.ml

```diff
@@ -128,20 +128,19 @@ let callv ?interrupt ?ssl_config uri reqs =
 let call ?interrupt ?ssl_config ?headers ?(chunked=false) ?(body=`Empty) meth uri =
   (* Create a request, then make the request. Figure out an appropriate
      transfer encoding *)
-  let req =
```

Comment:
I think it is easier to read if you don't pipe such a large block. I'd suggest
```
let (req, body') = match chunked with
[...]
in
req >>= request ?interrupt ?ssl_config ~body:body' ~uri
```

Reply:
I had another look at the code, similar blocks appear in other places. I am going to merge as is